packages/dev/core/src/Audio/sound.ts

```diff
@@ -351,15 +351,19 @@ export class Sound {
                                         this._htmlAudioElement.loop = this.loop;
                                         Tools.SetCorsBehavior(url, this._htmlAudioElement);
                                         this._htmlAudioElement.preload = "auto";
-                                        this._htmlAudioElement.addEventListener("canplaythrough", () => {
-                                            this._isReadyToPlay = true;
-                                            if (this.autoplay) {
-                                                this.play(0, this._offset, this._length);
-                                            }
-                                            if (this._readyToPlayCallback) {
-                                                this._readyToPlayCallback();
-                                            }
-                                        });
+                                        this._htmlAudioElement.addEventListener(
+                                            "canplaythrough",
+                                            () => {
+                                                this._isReadyToPlay = true;
+                                                if (this.autoplay) {
+                                                    this.play(0, this._offset, this._length);
+                                                }
+                                                if (this._readyToPlayCallback) {
+                                                    this._readyToPlayCallback();
+                                                }
+                                            },
+                                            { once: true }

```

Comment:
Added `{ once: true }` to avoid continuous calls on this event handler that occur in Chrome after `this._htmlAudioElement.currentTime` is set.

Reply:
it should still be removed when disposed

Reply:
So, add a call to `this._htmlAudioElement.removeEventListener` in `Sound.dispose`?

Reply:
yes, but you will need to keep a reference to the function to pass it to removeEvenetListenet

Reply:
I prefer to do this in a separate PR since this PR only added the `{ once: true }` option. The existing event listener not being disposed is an old issue.

Reply:
Sounds good! Removing the element from the HTML and GC-ing it will also clear this, so if that happens it is all good.

Reply:
Ok, cool. It gets removed from `document.body` in `dispose`, so we should be ok.

Reply:
Not enough for full garbage collection. if the sound is still referenced (not null-ed), the html element will still be available, so the event listener will still be registered.